spanner/unit_tests/test_client.py

```diff
@@ -106,6 +106,43 @@ def test_constructor_credentials_wo_create_scoped(self):
         expected_scopes = None
         self._constructor_test_helper(expected_scopes, creds)
 
+    def test_admin_api_lib_name(self):
+        """Establish that the lib_name and lib_version are passed to
+        the database and instance API objects in the GAPIC.
+        """
```

Comment:
Docstrings are about runtime discoverability, which is an actual downside for testcase methods:  they have caused testrunners to screw up display of failed tests in the past.  I have no idea whether current testrunners have this bug, because I resolved to avoid them forevermore after that, and use more descriptive method names and/or comments instead.